### ISSUE TITLE
Implement fixer for PLR0402

### ISSUE THREAD

**Issue by spaceone** [charliermarsh/ruff#2199]
A fixer for `PLR0402` would be very nice and probably relatively easy:
`import foo.bar as bar`
→
`from foo import bar`

**Comment by spaceone** [comment]
a quick workaround python script to achieve it:
```
import sys
import subprocess

lines = [line.split(':') for line in subprocess.check_output(['ruff', '-e', '--select', 'PLR0402', *sys.argv[1:]]).decode().splitlines()]
lines = [line for line in lines if len(line) == 4]
for filename, _, _, rest in lines:
    replacement = rest.split('`')[1]
    module = replacement.split(' ')[-1]
    package = replacement.split(' ')[1]
    subprocess.call(['sed', '-i', f's/import {package}.{module} as {module}/{replacement}/g', filename])
```

don't forget to execute `isort` afterwards.

**Comment by charliermarsh** [comment]
Closed by #2504.
